src/Microsoft.DotNet.Tools.Test/TestHostTracing.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.DotNet.Tools.Test
+{
+    public static class TestHostTracing
```

Comment:
We should consolidate our tracing APIs. Right now we use an ad-hoc combination of things, but we have `TraceSource` and [`Microsoft.Extensions.Logging`](https://www.github.com/aspnet/Logging) to consider.